src/Error/ErrorHandler.php

```diff
@@ -96,11 +98,9 @@ class ErrorHandler extends BaseErrorHandler
     public function __construct(array $options = [])
     {
         $defaults = [
-            'log' => true,
-            'trace' => false,
             'exceptionRenderer' => ExceptionRenderer::class,
         ];
-        $this->_options = $options + $defaults;
+        $this->_options = $options + $defaults + $this->_options;
```

Comment:
Should this class use `InstanceConfigTrait`?

Reply:
Yeah I guess making `BaseErrorHandler` use `InstanceConfigTrait` would be a good idea now.